content/applications/inventory_and_mrp/inventory/management/products/usage.rst

```diff
@@ -1,72 +1,243 @@
+==========================================
+Units of measure, packages, and packagings
+==========================================
 
-Units of Measure
+In Odoo, there are a variety of ways to specify the amount of products being bought, stocked, and
+sold. *Units of measure*, *packages*, and *packagings* are all available to streamline inventory
+flows, allowing for a variety of configurations for products as they enter and leave the warehouse.
+
+Units of measure
 ================
 
-Units of measure specify the unit used to handle a product. In Odoo, you
-have the possibility to specify the unit of measure in which you manage
-your stock and the one which is used when purchasing the product to your
-supplier.
+A *unit of measure (UoM)* refers to any of the different standards used to measure and handle a
+quanitifiable amount of products, such a units, weight, time, or size. Different :abbr:`UoMs (Units
+of Measure)` for weight, for example, can be kilos, pounds, ounces, grams, and so on.
+
+In Odoo, stock management and purchasing from suppliers are streamlined by specifying different
+:abbr:`UoMs (Units of Measure)` for both buying and selling products.
+
+.. image:: usage/uom-on-product-form.png
+   :align: center
+   :alt: Specify unit of measure for selling a product vs purchasing.
+
+Once a product has a default :guilabel:`Unit of Measure` and :guilabel:`Purchase Unit of Measure`
+set on the product form, Odoo automatically converts the different units in the product's
+purchase/sales orders and the corresponding delivery orders/receipts.
+
+The only condition is that all of the units have to be in the *same category* (unit, weight, volume,
+length, etc.).
 
-.. image:: usage/usage_01.png
-    :align: center
+.. example::
+   On the product form for `Rope`, the following fields are set as:
 
-The *conversion* between the different units of measure is done
-automatically. The only condition is that all the units have to be in
-the *same category* (Unit, Weight, Volume, Length,...)
+   - :guilabel:`Unit of Measure` in `ft` (feet), and
+   - :guilabel:`Purchase Unit of Measure` in `cm` (centimeters).
 
-For example, if I have the following reordering rule for the egg and I
-run the scheduler, the quantity added in the automatically generated
-purchase order will be in dozens but what will enter the stock will be
-units.
+   Because the vendor sells rope in `cm`, the purchase :guilabel:`UoM` is used to represent the
+   quantity on the :abbr:`PO (Purchase Order)`, which is also in centimeters.
 
-.. image:: usage/usage_02.png
-    :align: center
+   .. image:: usage/purchase-rope-in-cm.png
+      :align: center
+      :alt: Display purchase order for the product, rope, in centimeters.
 
-.. image:: usage/usage_03.png
-    :align: center
+After confirming the :abbr:`PO (Purchase Order)`, the quantity of product (found under the
+:guilabel:`Demand` column) is converted from the purchase :abbr:`UoM (Unit of Measure)` to the
```

Comment:
would more clearly indicate that **Demand** column doesn't appear unless the PO is confirmed first...

```suggestion
After confirming the :abbr:`PO (Purchase Order)`, the quantity of product (found under the
now visible :guilabel:`Demand` column) is converted from the purchase :abbr:`UoM (Unit of Measure)` to the
```